### DISCUSSION TITLE
Update Context Menu

### DISCUSSION THREAD

**Discussion by suleymansahin** [microsoft/vscode-discussions#491] [Extension Development QnA] [answered]
Hi, when we try to customize the context menu, it is possible to add new menu item, however we dont know how to disable default menu item. for example :  we want to disable/hide, '**Add Inline Breakpoint**' from right click menu, also from '**Run -> New Breakpoint -> Inline Breakpoint**' .
Thank you so much, have a nice day

**Comment by r3m0t** [comment, marked as answer]
Is that for a specific language?

Generally you can't remove or hide features, however for JSON it is not in the right click menu, so there must be a configuration causing that.

**Reply by suleymansahin** [reply]
for Cobol and Assembler, thank you so much for the answer
